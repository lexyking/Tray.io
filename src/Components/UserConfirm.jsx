import React, { Component } from 'react'


class userConfirm extends Component {

  continue = e => {
    e.preventDefault();
    this.props.nextStep();
  }

  back = e => {
    e.preventDefault();
    this.props.prevStep();
  }

  render() {
    const { name, email, role, isUpdate, isCommunication } = this.props.values;

    return (


      <section className="mainSection">
        <section className="formSection">
          <h2>Please review your information and click submit to continue !</h2>
          <section>
            <article>
              <h4>Name</h4>
              <p>{name}</p>
            </article>
            <article >
              <h4>Email</h4>
              <p>{email}</p>
            </article>
            <article >
              <h4>Role</h4>
              <p>{role}</p>
            </article>
            <article >
              <h4>Updates from Tray.io ?</h4>
              <p>{isUpdate ? 'Yes, i want to receive updates from Tray.io' : 'No, I do not wat to receive Updates'}</p>
            </article>
            <article >
              <h4>Do you want to receive communicational emails from Tray.io ?</h4>
              <p>{isCommunication ? 'Yes, i want to receive communicational emais from Tray.io' : 'No, I do not want to receive communicational emails from Tray.io'}</p>

            </article>

          </section>

          <button
            className="btn btn-back"
            onClick={this.back}
          >
            Back
            </button>

          <button
            className="btn"
            onClick={this.continue}
          >
            Continue & Confirm
            </button>
        </section>
      </section>


    )

  }
}


export default userConfirm
