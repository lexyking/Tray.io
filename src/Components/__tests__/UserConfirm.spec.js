import React from 'react';
import { create } from 'react-test-renderer';
import UserConfirm from '../UserConfirm';


describe('UserConfirm component', () => {
  test('Render Correctly', () => {
    const userConfirm = create(<UserConfirm values />);
    expect(userConfirm.toJSON()).toMatchSnapshot();
  })
})