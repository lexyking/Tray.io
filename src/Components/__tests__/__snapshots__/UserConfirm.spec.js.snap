// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserConfirm component Render Correctly 1`] = `
<section
  className="mainSection"
>
  <section
    className="formSection"
  >
    <h2>
      Please review your information and click submit to continue !
    </h2>
    <section>
      <article>
        <h4>
          Name
        </h4>
        <p />
      </article>
      <article>
        <h4>
          Email
        </h4>
        <p />
      </article>
      <article>
        <h4>
          Role
        </h4>
        <p />
      </article>
      <article>
        <h4>
          Updates from Tray.io ?
        </h4>
        <p>
          No, I do not wat to receive Updates
        </p>
      </article>
      <article>
        <h4>
          Do you want to receive communicational emails from Tray.io ?
        </h4>
        <p>
          No, I do not want to receive communicational emails from Tray.io
        </p>
      </article>
    </section>
    <button
      className="btn btn-back"
      onClick={[Function]}
    >
      Back
    </button>
    <button
      className="btn"
      onClick={[Function]}
    >
      Continue & Confirm
    </button>
  </section>
</section>
`;
